### PR TITLE
More PSRAM and ELECROW support 

### PR DESCRIPTION
**Pull request by davepl**
Makes more use of PSRAM, particularly in vectors.  Adds elecrow display support.  Fixes FPS bug.

* [X] I read the contribution guidelines in [CONTRIBUTING.md](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/CONTRIBUTING.md).
* [X] I understand the BlinkenPerBit metric, and maximized it in this PR.
* [X] I selected `main` as the target branch.
* [X] All code herein is subjected to the license terms in [COPYING.txt](http://github.com/PlummersSoftwareLLC/NightDriverStrip/blob/main/COPYING.txt).

### CODE REVIEW

**Comment by davepl** [comment]
You could add code to override it, if you like, but prefer some default baked in there so people (a) know where to replace the IP, and (b) its easy to run multiple times.  But no objection to making it an argument if you can still run it from VS Code easily.

- Dave

File: samples/audioserver/audioserver.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 # NightDriver ESP32 wifi address - update to your ESP32 WiFi
 
-client = '192.168.8.250'        
+client = '192.168.8.47'
```
[the remark this answers is not in the document]
Comment (davepl) [reply samples/audioserver/audioserver.py:26]: Not for this checkin, but a reasonable idea!

[the remark this answers is not in the document]
Comment (rbergen) [reply samples/audioserver/audioserver.py:26]: My two cents about this: this being a sample that's written in Python and intended at least to some extent to show how to code "towards" the project, I think setting the target IP address this way is defendable. It invites the user of the sample to engage with the code in it.

**Review by rbergen — APPROVED** [review]
LGTM!

**Pull request merged**